feat(2025): use DevSummit 2025 template

maxpatiiuk opened the pull request:
- Update slidev templates to use the DevSummit 2025 templates now that they are released.
- Update slidev formatting to match templates as closely as possible.
- Removed the starter slidev presentation - it is present in the git history if you need to get back to it for reference

For future reference:
- For styling, can create separate CSS files, or can add styles inside of markdown using [UnoCSS](https://unocss.dev/) (tailwind-like library, from Vue team, enabled by default in Slidev)
  - Scoped styles example: https://sli.dev/features/slide-scope-style
- For common layouts, pre-defined styles are available: https://sli.dev/builtin/layouts

maxpatiiuk committed:
feat(2025): use DevSummit 2025 template

omarkawach reviewed (approved):
Nice work on implementing the template in slidev :)

Is there anything that can be done about the flickering when quickly switching between slides?

![slidev](https://github.com/user-attachments/assets/8fc7df17-8c70-4bc9-aaf1-e8314d633c17)

I saw this flickering when running the `start` script and when I run `preview` (I added `"preview": "vite preview"` locally to see if it was only a thing in dev)

If I wait more than 10 seconds per slide then I don't see the flickering issue.

In order to see the slidev as a PDF, I had to resolve this error

![image](https://github.com/user-attachments/assets/964e6fd0-bda2-4fbd-a9fb-4655f64c0d82)

maxpatiiuk committed:
perf(2025): preload images for future slides

maxpatiiuk committed:
build(2025): add preview script

maxpatiiuk committed:
fix(2025): make export to PDF work

maxpatiiuk committed:
feat(2025): make socials links clickable

maxpatiiuk commented:
> Is there anything that can be done about the flickering when quickly switching between slides?

Fixed. Images will now be loaded ahead of time

> I added "preview": "vite preview" locally

Thanks for suggestion - added it to package.json

> In order to see the slidev as a PDF, I had to resolve this error

Fixed this and other PDF export issues

Thanks for review!
